app/reducers.js

```diff
@@ -8,6 +8,7 @@ import { connectRouter } from 'connected-react-router';
 import history from 'utils/history';
 import globalReducer from 'containers/App/reducer';
 import languageProviderReducer from 'containers/LanguageProvider/reducer';
+import testNicolaReducer from './containers/TestNicola/reducer';
```

Comment:
Anche questo non dovrebbe servirti, in index.js del tuo container dovrebbe esserci un hook injectReducer che fa la stessa cosa.